src/System.Linq.Parallel/tests/Combinatorial/CancellationParallelQueryCombinationTests.cs

```diff
@@ -33,6 +33,11 @@ public static void Aggregate_AggregateException_Wraps_OperationCanceledException
             Functions.AssertAggregateAlternateCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate(0, (i, j) => j, i => i));
             Functions.AssertAggregateAlternateCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate(0, (i, j) => j, (i, j) => i, i => i));
             Functions.AssertAggregateAlternateCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate(() => 0, (i, j) => j, (i, j) => i, i => i));
+            Functions.AssertAggregateNotCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate((i, j) => j));
```

Comment:
I need to work on my naming.  
Alternate in this case is a different-source canceled token, while not canceled is throwing with the same token, but it's not canceled.  This should be two different branches, but it might be lower than most of this needs to worry about, will check.